pynest/nest/tests/test_parrot_neuron.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+#
+# test_parrot_neuron.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+# This script tests the parrot_neuron in NEST.
+
+import nest
+import unittest
+
+
+@nest.check_stack
+class ParrotNeuronTestCase(unittest.TestCase):
+    """Check parrot_neuron spike repetition properties"""
+
+    def setUp(self):
+        nest.set_verbosity(100)
```

Comment:
I would not recommend shutting the kernel up so brutally. A bit of output makes it easier to find out what went wrong when a test fails. `nest.set_verbosity('M_WARNING')` should do fine.